pyproject.toml

```
@@ -27,7 +27,9 @@ isort = "^5.7"
 yapf = ">=0.32"
 toml = "*"
 mypy = "^1.2.0"
-pandas-stubs = "^1.5.3.230321"
+pandas-stubs = "*"
+pyyaml = "6.0.1"
```

[question]

new dependency...?

---

```diff
 name = "pyyaml"
-version = "6.0"
+version = "6.0.1"
```

seems not new.

`6.0.0` has any issue? I expect we don't need to pin but just require newer than 6.0.1

```suggestion
pyyaml = ">=6.0.1"
```